lib/client.js

```diff
@@ -352,20 +351,12 @@ export default class Client {
       this.ready = true
       this._metadata = data.metadata
       this._context = data.context
-      this._iframe_session_timeout = data.iframe_session_timeout
+
       this._idleState = new IdleState(data.iframe_session_timeout)
```

Comment:
Think you can also remove the `data.iframe_session_timeout`

Reply:
✅